### ISSUE TITLE
Emoji misclicks

### ISSUE THREAD

**Issue by Aidar** [izyumkin/MCEmojiPicker#37]
When scrolling the keyboard, it is quite common to select emoji although I didn't want it. I would like the emoji to be selected only when tapping on it. Any scroll should prevent selecting. 

Overall this is a great framework, thanks!
P.S. Waiting for search 🙏

**Comment by truthhonestmessaging** [comment]
Yeah this is the serious & biggest issue with this library. I'm thinking of importing files so I edit this library on my own
